libs/utils/utils.js

```diff
@@ -437,6 +438,14 @@ function decorateSections(el, isDoc) {
   });
 }
 
+async function buildTagsBlock() {
```

Comment:
I'd opt for a template to do this rather than polluting utils.

Reply:
@auniverseaway 
As far as I know, to load a template the `metadata` table should contain `|template |tags|` like:
| Metadata   ||  
|----------|:-------------:|
| Template |  Tags |
| Author |  John Doe |
| Tags |  Creative Cloud, Adobe Life |

Is that correct?

If so, we could use template for `tags` instead of making it a `block`.

Reply:
Yeah.